### PR TITLE
580 thank you

### PR DESCRIPTION
**Pull request by lschatzkin**
Styling, social share links and all things thank you.

### CODE REVIEW

File: app/helpers/action_page_helper.rb [hunk 1]
```diff
@@ -38,6 +38,9 @@ def email_friends_url(action_page)
     "mailto:?subject=#{u subject}&body=#{u message}"
   end
 
+  def link_share_url(action_page)
```
Comment (vbrown608) [inline app/helpers/action_page_helper.rb]: What's the purpose of this helper? Seems like maybe we're getting the share url in javascript in `copy-url.js`.

File: app/views/tools/_petition.html.erb [hunk 2]
```diff
@@ -104,7 +104,9 @@
     </div>
   </div>
 </div>
-
+</div>
+</div>
+</div>
```
Comment (vbrown608) [inline app/views/tools/_petition.html.erb]: What's the reason for these additional closing div tags? It looks like they might not have matching opening tags in the template.

File: app/views/tools/_thank_you.html.erb [hunk 3]
```diff
@@ -2,27 +2,36 @@
...
+      <hr>
+      <p> EFF is supported by donations from folks like you. We appreciate anything you can give! </p>
+      <a class="eff-red-button" href="https://supporters.eff.org/donate">Donate</a>
+      <% if @actionPage.enable_tweet && @actionPage.tweet %>
       <% if @actionPage.tweet.targets.present? && (@actionPage.tweet.targets.size > 1) %>
-        <span id="tweet-back">Want to tweet at others? Go back</span>
+      <span id="tweet-back">Want to tweet at others? Go back</span>
+      <% end %>
       <% end %>
-    <% end %>
```
Comment (vbrown608) [inline app/views/tools/_thank_you.html.erb]: Indenting within `if` statements would be helpful for readability. Something like:
```
      <% if @actionPage.enable_tweet && @actionPage.tweet %>
        <% if @actionPage.tweet.targets.present? && (@actionPage.tweet.targets.size > 1) %>
          <span id="tweet-back">Want to tweet at others? Go back</span>
          <span id="tweet-back">Want to tweet at others? Go back</span>
        <% end %>
      <% end %>
```

File: app/helpers/action_page_helper.rb (hunk 1 above)
Comment (lschatzkin) [reply app/helpers/action_page_helper.rb]: You're right, that's a remnant of another thought path I went down.

File: app/views/tools/_petition.html.erb (hunk 2 above)
Comment (lschatzkin) [reply app/views/tools/_petition.html.erb]: That was a hack I forgot to remove, removing now.

File: app/views/tools/_thank_you.html.erb (hunk 3 above)
Comment (lschatzkin) [reply app/views/tools/_thank_you.html.erb]: You speak the truth, my friend.

**Pull request merged**